app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+  end  
+
+  def update
```

Comment:
Using `#update` as the action here would make sense if we're altering the attributes of a row in the `movies` table - but what we're really doing is _creating a relationship_ between a specific Movie and an Actor by its ID. That relationship is defined by the `movie_actors` table, so a few things to consider: 

1. Does a `create` method make more sense? Yes, but we're not really creating a movie, so does that method belong in this controller?
2. Consider moving a `create` method to a controller called `movie_actors_controller.rb` instead.